codemods/1.x-2.x/__testfixtures__/computed-decorator.output.js

```diff
@@ -0,0 +1,14 @@
+import {connect as Cerebral} from 'cerebral/react';
```

Comment:
https://github.com/cerebral/cerebral/blob/391a142da89628938c0c31263525d7aa084f3a3e/packages/demo/src/components/Tasks/index.js

Looking at this demo, I'd say it's still call connect.

```
@connect()
class ....
```

is the same as

```
connect(
  class ...
)
```

I just rename it to `@Cerebral` :)